### DISCUSSION TITLE
Paths information

### DISCUSSION THREAD

**Discussion by Zhong0501** [NVlabs/sionna#648] [General]
Hi, I'm having some problems with the paths I'm getting from Sionna's RT simulation, is the angle information (e.g. phi_r) for each path expressed in radians or in angular system? Is the magnitude information (a) of each path a linear value or a dB value?

**Comment by faycalaa** [comment]
Hi @Zhong0501,

The angles of arrival and departure are in radians, and the channel coefficients [Paths.a](https://nvlabs.github.io/sionna/api/rt.html#sionna.rt.Paths.a) are complex-valued. The amplitudes of these coefficients correspond to the path loss in linear scale.

**Reply by potatoess11** [reply]
Hi,
When the maximum number of reflections is set to 2 and above, I wonder if the angle information in paths corresponds to the relevant angle at which the last hop reaches the receiver or something else

**Comment by SebastianCa** [comment]
Moving this thread to discussions.
